content/ReleaseNotes/TrueCommand/2.1.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "2.1"
+weight: 8
+---
+
+**January 4, 2022**
+
+The TrueCommand team is pleased to announce [TrueCommand 2.1](https://www.truenas.com/docs/truecommand/) is now available. 
+
+TrueCommand 2.1 is the single pane of glass for:
+
+* **TrueNAS CORE**: Manage systems on standard servers, Minis, or even AWS.
+* **TrueNAS Enterprise**: Manage X-Series and M-Series systems with High Availability.
+* **TrueNAS SCALE**: Manage a group of systems running a TrueNAS SCALE cluster. 
+
+## 2.1 Changelog
+
+### New Feature
+
+<ul>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1184'>TC-1184</a>] -         Add two-factor authentication support
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1581'>TC-1581</a>] -         Have TC auto-generate and use an auth token after initial NAS connect
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1711'>TC-1711</a>] -         NAS user management
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1757'>TC-1757</a>] -         Add SAML SSO support
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1774'>TC-1774</a>] -         Add ability to manage NAS users/groups for shares.
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1823'>TC-1823</a>] -         Add ability to reset user passwords from login page
+</li>
+</ul>
+
+### Improvement
+
+<ul>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1468'>TC-1468</a>] -         EULA needs to identify GPL components
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1489'>TC-1489</a>] -         Question about the Dashboard System Options Menu
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1603'>TC-1603</a>] -         Update selenium tests
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1655'>TC-1655</a>] -         Include "Group" or "All" option for system selection for reports
```

Comment:
Best practices is to Never use " " for emphasis, if a UI element use bold if exactly as it appears on-screen change Group and All to bold.